functions/Set-DbaAgentAlert.ps1

```diff
@@ -149,9 +150,9 @@ function Set-DbaAgentAlert {
             #endregion alert options
 
             # Execute
-            if ($PSCmdlet.ShouldProcess($SqlInstance, "Changing the alert $a")) {
+            if ($PSCmdlet.ShouldProcess($SqlInstance, "Committing changes for alert $a")) {
                 try {
-                    Write-Message -Message "Changing the alert" -Level Verbose
+                    Write-Message -Message "Committing changes for alert $a" -Level Verbose
```

Comment:
You don't need this line, as ShouldProcess also has verbose output. (But if you still like to have the line: Like in line 135, this should not be `$a` but `$currentAlert`.)